.gitignore

```diff
@@ -8,4 +8,7 @@ test.py
 docs_crew/
 chroma.sqlite3
 venv
-.venv/
\ No newline at end of file
+.venv/
+
+# chroma db
+db
```

Comment:
if you're deploying a crew, don't you want this ?

Reply:
good call! 
If we’re using a local DB, then yes, it will. I'm going to drop it.

Reply:
actually, I have added db folder to this repo’s .gitignore, so it shouldn’t affect any other repositories